string station_id
float64 distance
---
geometry_msgs/Pose inbound
geometry_msgs/Pose outbound
geometry_msgs/Pose queue
